src/typedefs/UseAssetsResult.ts

```diff
@@ -2,8 +2,8 @@ import type { UseAssetsStatus } from '../constants/UseAssetsStatus';
 
 export interface UseAssetsResult<T>
 {
-    /** @description An array of resolved assets, or `null` for assets that are still loading. */
-    assets: (T | null)[];
+    /** @description An array of resolved assets, or `undefined` for assets that are still loading. */
+    assets: (T | undefined)[];
```

Comment:
Previously, this was null, and that may have been intentional. However, it required that all assets be explicitly checked for against null before being passed as a prop. If the intention was to use the isSuccess boolean for conditional rendering, then this being undefined by default makes more sense IMO.